src/os/config/configinstance.js

```diff
@@ -0,0 +1,35 @@
+goog.module('os.config.instance');
```

Comment:
Is this singleton pattern necessary for `Settings` (and by extension, all singletons, even if they don't pose major circular dependency issues?) With settings in particular we use it EVERYWHERE, and switching them all over just seems kind of annoying.

Reply:
I'm only creating these in places where a circular dependency exists, so it's helpful to have a way to reference the singleton without a hard dependency. Using it isn't expected/required, unless it resolves a circular dep.

Reply:
Gotcha. Wouldn't have expected any circular dependencies with settings since stuff usually just requires it in and it sits at the top of any dependency tree and doesn't require too much stuff itself.

Reply:
The cycle was between `os.config` and `os.config.Settings` (uses appNs/coreNs). I didn't want to move anything in `os.config` because it's used all over the place, so I created the instance module instead.